src/proxy/query.go

```diff
@@ -105,18 +105,21 @@ func (spanner *Spanner) ComQuery(session *driver.Session, query string, bindVari
 	defer func() {
 		queryStat(node, timeStart, slowQueryTime, err)
 	}()
+	status := uint32(0)
```

Comment:
If add a comment here to explain the meaning of `status` and where it is used, this will be better for code reading.